packages/remix-themes/src/theme-provider.tsx

```diff
@@ -67,11 +95,13 @@ export function ThemeProvider({
     })
 
     broadcastThemeChange(theme)
+    disableTransitionOnThemeChange && disableTransition()
```

Comment:
Hey @FedDiSte , thanks for this PR. 

`disableTransition` function seems to return a cleanup function where the styles are removed from the body. But I don't see where this cleanup function is invoked.

Reply:
Yeah i just saw that, i forgot to remove that, i adapted the code i was using in my project, i used that function in a different way, i will commit a fix later

Reply:
@abereghici i've just updated the correct code, sorry for the wrong commit

Reply:
@FedDiSte looks good! thank you! 
I do have a question for this line `(() => window.getComputedStyle(document.body))();` why it should be wrapped in an IIFE?

Reply:
@abereghici I used that just to force the restyle of the body, it could be outside the IIFE too